### DISCUSSION TITLE
Lambda Annotations: .Net Core 8 with NSwag

### DISCUSSION THREAD

**Discussion by AadiGitt** [aws/aws-lambda-dotnet#1744] [Q&A] [answered]
I went over https://towardsaws.com/access-swagger-ui-in-aws-lambda-via-api-gateway-deployed-with-the-serverless-framework-d80268ef1d79 document but this wouldn't help with .net 8 and Lambda Annotations. Any help would be appreciated. Or perhaps help in creating swagger json/yml document which can then be later consumed by Redoc/NSwag.

**Comment by ashishdhingra** [comment, marked as answer]
@AadiGitt We have an investigative backlog item for the similar issue and would update accordingly.
